static/app/views/replays/list/replaySearchBar.tsx

```diff
@@ -50,33 +52,69 @@ function fieldDefinitionsToTagCollection(fieldKeys: string[]): TagCollection {
 
 const REPLAY_FIELDS_AS_TAGS = fieldDefinitionsToTagCollection(REPLAY_FIELDS);
 const REPLAY_CLICK_FIELDS_AS_TAGS = fieldDefinitionsToTagCollection(REPLAY_CLICK_FIELDS);
+const EXCLUDED_TAGS = ['browser', 'device', 'os', 'user'];
```

Comment:
why are these excluded?

Reply:
we already have browser.name, device.name, etc. They're effectively the same

Reply:
maybe user is actually user.id, but either way they're confusing

Reply:
ohh gotcha, could we add a comment to clarify that? just for posterity